clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -135,9 +135,11 @@ export default async function handler(
   const script = `
   (function () {
     // This polyfill service adds a fetch polyfill only when needed, depending on browser making the request 
-    var script = document.createElement('script');
-    script.src = 'https://polyfill.io/v3/polyfill.min.js?features=fetch';
-    document.head.appendChild(script);
+    if (window.fetch) {} else {
```

Comment:
is there a reason to not do the negation instead? i.e.

```javascript
if (!window.fetch) {
  var script = ...
}
```

Reply:
nah, I'll reverse this logic!